### PR TITLE
Modify vfx and llpc include directory in cmake file

### PR DESCRIPTION
**Pull request by xuechen417**
vfx and llpc include directory will be moved out of llpc subdirectory.

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit efd7878710bc7342b459781ca6ad6873e904da1f
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>LLPC: dev b628de8913ff7ae7f8013a358a147e3ab4b10fd9</li>
<li>PAL: dev 9fab16015e522fff05890a045a1e9d8d3c23a636</li>
<li>XGL: dev 052ebfa2e77abd14fdb4f5120b35b081f3fc4bdc</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 1ecf0a1496f32eaf34825b3d11432c6b1d03c951</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 942/942 (100.0%)</li>
<li>Failed: 0/942 (0.0%)</li>
</ul></details>

File: CMakeLists.txt [hunk 1]
```diff
@@ -66,24 +66,36 @@ add_library(spvgen SHARED "")
 
 set_target_properties(spvgen PROPERTIES PREFIX "")
 
+if(EXISTS ${PROJECT_SOURCE_DIR}/../llpc/tool/vfx)
+	set(XGL_VFX_PATH ${PROJECT_SOURCE_DIR}/../llpc/tool/vfx CACHE PATH "The path of vfx.")
```
Comment (JacobHeAMD) [inline CMakeLists.txt]: alignment

**Comment by amdrexu** [comment]
I have no comments. Just fix Jacob's comments.

File: CMakeLists.txt (hunk 1 above)
Comment (xuechen417) [reply CMakeLists.txt]: Fixed.

**Comment by amdvlk-admin** [comment]
### Test summary for commit 278ab7f7f4fcf380c343328eafa41d4b309a0bc1
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>LLPC: dev b628de8913ff7ae7f8013a358a147e3ab4b10fd9</li>
<li>PAL: dev 9fab16015e522fff05890a045a1e9d8d3c23a636</li>
<li>XGL: dev 052ebfa2e77abd14fdb4f5120b35b081f3fc4bdc</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 1ecf0a1496f32eaf34825b3d11432c6b1d03c951</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 942/942 (100.0%)</li>
<li>Failed: 0/942 (0.0%)</li>
</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 4d8220122dab45841ebf0f591ff90e86e54eb9f5
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>LLPC: dev b628de8913ff7ae7f8013a358a147e3ab4b10fd9</li>
<li>PAL: dev 9fab16015e522fff05890a045a1e9d8d3c23a636</li>
<li>XGL: dev 052ebfa2e77abd14fdb4f5120b35b081f3fc4bdc</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 1ecf0a1496f32eaf34825b3d11432c6b1d03c951</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 942/942 (100.0%)</li>
<li>Failed: 0/942 (0.0%)</li>
</ul></details>

**Pull request merged**